typescript/nomad-deploy/scripts/calculateDomain.ts

```diff
@@ -0,0 +1,10 @@
+import { getHexDomainFromString } from '@nomad-xyz/sdk/utils';
+
+// npm run calculate-domain <network abbreviation (e.g. "eth")>
+(async () => {
+  const args = process.argv.slice(2);
+  const network = args[0];
+
+  const hexBytes = getHexDomainFromString(network);
+  console.log(`${network} interpreted as hex number: ${hexBytes}`);
+})();
```

Comment:
This is util script for future use when we need to calculate mainnet domains from network abbreviations